cpp/src/io/comp/nvcomp_adapter.hpp

```diff
@@ -75,20 +75,12 @@ size_t batched_decompress_temp_size(compression_type compression,
                                                     uint32_t max_uncomp_chunk_size);
 
 /**
- * @brief Gets input alignment requirements for the given compression type.
+ * @brief Gets input and output alignment requirements for the given compression type.
  *
  * @param compression Compression type
- * @returns required alignment, in bits
+ * @returns required alignment
  */
-[[nodiscard]] size_t compress_input_alignment_bits(compression_type compression);
-
-/**
- * @brief Gets output alignment requirements for the given compression type.
- *
- * @param compression Compression type
- * @returns required alignment, in bits
- */
-[[nodiscard]] size_t compress_output_alignment_bits(compression_type compression);
+[[nodiscard]] size_t required_alignment(compression_type compression);
```

Comment:
Likely a silly question:  Is it no longer possible for input and output alignment to diverge for the same compression type?

Reply:
I'm just using the nvcomp API as intended :)
I do expect that the actual requirements differ and the values here are max(in_req, out_req).
CC @eschmidt-nvidia for a more definitive answer :)

Reply:
Yes, that's right. It's possible we'll add more variables to specify alignment for in / out req separately.